_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Installing OpenSearch Benchmark
+nav_order: 5
+has_children: false
+---
+
+# Installing OpenSearch Benchmark
+
+You can install OpenSearch Benchmark directly on a host running Linux or macOS or you can run OpenSearch Benchmark in a Docker container on any compatible host. This page describes high-level considerations for your OpenSearch Benchmark host, as well as instructions for installing OpenSearch Benchmark.
+
+## Docker limitations
+
+Some OpenSearch Benchmark functionality is unavailable when you run OpenSearch Benchmark in a Docker container. Specifically, the following restrictions apply:
+- You cannot distribute load worker coordinator hosts.
+- You can only use the `benchmark-only` pipeline.
```

Comment:
It might be nice to elaborate on what the two of these mean. New-comers might not entirely understand this unless they play around with OSB. We're also planning to rename `load-worker-coordinator-hosts` in the future as it's a mouthful and doesn't offer much insight into what it is. Perhaps something like:
```
- OSB cannot distribute load from multiple hosts (i.e. cannot use load worker coordinator hosts)
- OSB cannot provision OpenSearch nodes and can only run tests on pre-existing clusters (i.e. can only invoke OSB commands using `benchmark-only` pipeline)
```
There's probably a better way to say it but just an idea!